# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * scheduler_error_mailer
# 
# Translators:
# Rudolf Schnapka <rs@techno-flex.de>, 2016
msgid ""
msgstr ""
"Project-Id-Version: server-tools (8.0)\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2016-01-19 14:44+0000\n"
"PO-Revision-Date: 2016-01-18 14:02+0000\n"
"Last-Translator: Rudolf Schnapka <rs@techno-flex.de>\n"
"Language-Team: German (http://www.transifex.com/oca/OCA-server-tools-8-0/language/de/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: de\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#. module: scheduler_error_mailer
#: model:email.template,body_html:scheduler_error_mailer.scheduler_error_mailer
msgid ""
"\n"
"<div style=\"font-family: 'Lucica Grande', Ubuntu, Arial, Verdana, sans-serif; font-size: 12px; color: rgb(34, 34, 34); background-color: #FFF; \">\n"
"\n"
"<p>OpenERP tried to run the scheduler <em>${object.name or ''}</em> in the database <em>${ctx.get('dbname')}</em> but it failed. Here is the error message :</p>\n"
"\n"
"<strong>\n"
"${ctx.get('job_exception') and ctx.get('job_exception').value or 'Failed to get the error message from the context.'}\n"
"</strong>\n"
"\n"
"<p>You may check the logs of the OpenERP server to get more information about this failure.</p>\n"
"\n"
"<p>Properties of the scheduler <em>${object.name or ''}</em> :</p>\n"
"<ul>\n"
"<li>Model : ${object.model or ''}</li>\n"
"<li>Method : ${object.function or ''}</li>\n"
"<li>Arguments : ${object.args or ''}</li>\n"
"<li>Interval : ${object.interval_number or '0'} ${object.interval_type or ''}</li>\n"
"<li>Number of calls : ${object.numbercall or '0'}</li>\n"
"<li>Repeat missed : ${object.doall}</li>\n"
"<li>User : ${object.user_id.name or ''}</li>\n"
"</ul>\n"
"\n"
"<p>\n"
"-- <br/>\n"
"Automatic e-mail sent by OpenERP. Do not reply.<br/>\n"
"Database : ${ctx.get('dbname')}\n"
"</p>\n"
"</div>\n"
"    "
msgstr "\n<div style=\"font-family: 'Lucica Grande', Ubuntu, Arial, Verdana, sans-serif; font-size: 12px; color: rgb(34, 34, 34); background-color: #FFF; \">\n\n<p>Odoo hat versucht die Terminaufgabe <em>${object.name or ''}</em> in der Datenbank <em>${ctx.get('dbname')}</em> auszuführen, scheiterte jedoch. Hier folgt die Fehlermeldung:</p>\n\n<strong>\n${ctx.get('job_exception') and ctx.get('job_exception').value or 'Konnte Fehlermeldung nicht aus dem Kontext erfahren.'}\n</strong>\n\n<p>Sie können die Logs des Odoo-Server überprüfen, um weitere Information zu diesem Fehler zu erhalten.</p>\n\n<p>Eigenschaften der Terminaufgabe <em>${object.name or ''}</em> :</p>\n<ul>\n<li>Modell: ${object.model or ''}</li>\n<li>Methode: ${object.function or ''}</li>\n<li>Argumente: ${object.args or ''}</li>\n<li>Intervall: ${object.interval_number or '0'} ${object.interval_type or ''}</li>\n<li>Anzahl Aufrufe: ${object.numbercall or '0'}</li>\n<li>Verpaßte nachholen: ${object.doall}</li>\n<li>Benutzer: ${object.user_id.name or ''}</li>\n</ul>\n\n<p>\n-- <br/>\nAutomatisch versandte Email aus Odoo. Nicht beantworten.<br/>\nDatenbank : ${ctx.get('dbname')}\n</p>\n</div>\n    "

#. module: scheduler_error_mailer
#: code:addons/scheduler_error_mailer/ir_cron.py:75
#, python-format
msgid "Error :"
msgstr "Fehler:"

#. module: scheduler_error_mailer
#: field:ir.cron,email_template_id:0
msgid "Error E-mail Template"
msgstr "Fehler-Email-Vorlage"

#. module: scheduler_error_mailer
#: help:ir.cron,email_template_id:0
msgid "Select the email template that will be sent when this scheduler fails."
msgstr "Wählen Sie die Email-Vorlage, die gesendet werden soll, wenn diese Terminaufgabe scheitert."

#. module: scheduler_error_mailer
#: code:addons/scheduler_error_mailer/ir_cron.py:76
#, python-format
msgid "Task failure with UID = %d."
msgstr "Aufgabenfehler mit UID = %d."

#. module: scheduler_error_mailer
#: model:email.template,subject:scheduler_error_mailer.scheduler_error_mailer
msgid "[DB ${ctx.get('dbname')}] Scheduler '${object.name or ''}' FAILED"
msgstr ""
